### PR TITLE
feat(matrices): allow csv import

### PR DESCRIPTION
**Pull request by MartinBelthle**
Fix [ANT-1588]

### CODE REVIEW

File: tests/integration/raw_studies_blueprint/test_fetch_raw_data.py [hunk 1]
```diff
@@ -57,7 +57,8 @@ def test_get_study(
         with db():
             study: RawStudy = db.session.get(Study, internal_study_id)
             study_dir = pathlib.Path(study.path)
-        headers = {"Authorization": f"Bearer {user_access_token}"}
+        client.headers = {"Authorization": f"Bearer {user_access_token}"}
+        raw_url = f"/v1/studies/{internal_study_id}/raw"
```
Comment (maugde) [inline tests/integration/raw_studies_blueprint/test_fetch_raw_data.py:61]: good improvement

File: tests/integration/raw_studies_blueprint/test_fetch_raw_data.py [hunk 2]
```diff
@@ -134,39 +120,40 @@ def test_get_study(
...
-            headers=headers,
-        )
+        res = client.get(raw_url, params={"path": "user/somewhere/something.txt"})
         assert res.status_code == 200, res.json()
         assert res.content == b"This is the end!"
 
+        # You can import a csv or a tsv file inside a matrix
+        matrix_path = "input/load/series/load_de"
+        for content, delimiter in zip(
+            [b"1.20000\n3.400000\n", b"1.4\t0.5\n0.0\t0.4\n", b"1.2,1.3\n1.4,1.5\n"], ["\t", "\t", ","]
```
Comment (maugde) [inline tests/integration/raw_studies_blueprint/test_fetch_raw_data.py:146]: I don't know if it could be useful to also test with semicolons here

**Review by maugde — COMMENTED** [review]
It seems good for me. Only have one question about the integration test.

File: tests/integration/raw_studies_blueprint/test_fetch_raw_data.py (hunk 2 above)
Comment (MartinBelthle) [reply tests/integration/raw_studies_blueprint/test_fetch_raw_data.py:146]: I could have done it indeed. If someone encounters a problem we'll add tests here indeed.

**Pull request merged**